### PR TITLE
Code refactoring with pylint

### PR DESCRIPTION
**Pull request by ostapbodnar**

### CODE REVIEW

**Review by irayarka — APPROVED** [review]

File: pytest/test_another_algo.py [hunk 1]
```diff
@@ -293,25 +296,86 @@ def test_task178d(input_value, expected_value):
...
+                5,
+                10,
+                10,
+                5,
+                1,
+                "\n",
+            ],
+        ),
+    ],
+)
```
Comment (BurlyLlama28) [inline pytest/test_another_algo.py:369]: promise to fix it, because it's scary

**Review by BurlyLlama28 — APPROVED** [review]

File: pytest/test_another_algo.py (hunk 1 above)
Comment (ostapbodnar) [reply pytest/test_another_algo.py:369]: I will try)

File: requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ pytest==6.2.2
 parameterized==0.8.1
 pre-commit
 pylint==2.7.2
+parameterized
```
Comment (Kavjar) [inline requirements.txt:5]: What is it?

**Review by Kavjar — APPROVED** [review]
Well done!!!

**Pull request merged**